RIGS/templates/RIGS/eventauthorisation_client_request.txt

```diff
@@ -0,0 +1,12 @@
+Hi there,
+
+{{request.user.get_full_name}} has requested that you authorise N{{object.pk|stringformat:"05d"}} | {{object.name}}.
+
+Please find the link below to complete the event booking process.
+{% if object.event.organisation and object.event.organisation.union_account %}{# internal #}
+Remember that only Presidents or Treasurers are allowed to sign off payments. You may need to forward this email on.
+{% endif %}
+
+{{request.scheme}}://{{request.get_host}}{% url 'event_authorise' object.pk hmac %}
+
+The TEC Rig Information Gathering System
```

Comment:
I always think that looks god awful, so I've set the reply-to address to something sensible now so theoretically we should never have a problem.

I would also suggest setting rigs@nottinghamtec.co.uk to forward somewhere sensible (productions or it, one or the other)